compiler/src/test/java/com/squareup/anvil/compiler/dagger/AssistedInjectGeneratorTest.kt

```diff
@@ -368,7 +368,7 @@ public final class AssistedService_Factory {
         .invoke(null, Provider { 5 })
       assertThat(factoryInstance::class.java).isEqualTo(factoryClass)
 
-      val newInstance = staticMethods.single { it.name == "newInstance" }
+      staticMethods.single { it.name == "newInstance" }
```

Comment:
ugh sorry!

Reply:
Not your fault, the check shouldn't have been broken in the first place.